sqlitecluster/SQLiteClusterMessenger.h

```diff
@@ -0,0 +1,20 @@
+#include <libstuff/libstuff.h>
+#include <libstuff/SHTTPSManager.h>
+
+class SQLiteNode;
+class BedrockCommand;
```

Comment:
ah interesting -  should we add this to our auth style guide as a recommended best practice then? I've probably been doing it the opposite myself so probably added to the problem at some point 😬

Reply:
Would you mind adding an inline comment explaining this?

Reply:
Why here instead of all these other places we already do this?

```
SynchronizedMap.h
11:    friend class LockGuard;

test/lib/BedrockTester.h
7:class SQLite;

plugins/Jobs.h
6:  friend class BedrockJobsCommand;

libstuff/STCPManager.h
12:class SSSLState;
13:class SX509;

libstuff/SHTTPSManager.h
6:class BedrockPlugin;

libstuff/libstuff.h
25:class SQResult;
26:class SFastBuffer;
27:class SData;

libstuff/SSSLState.h
10:class SX509;
11:class SFastBuffer;

libstuff/STCPNode.h
136:        friend class STCPNode;
137:        friend class SQLiteNode;

BedrockCommand.h
5:class BedrockPlugin;
138:        friend class BedrockCommand;

sqlitecluster/SQLiteCore.h
2:class SQLite;

sqlitecluster/SQLiteNode.h
9:class SQLiteCommand;
10:class SQLiteServer;
15:    friend class SQLiteNodeTester;

sqlitecluster/SQLiteServer.h
2:class SQLiteCommand;

sqlitecluster/SQLiteUtils.h
5:class SQLite;

BedrockCore.h
4:class BedrockServer;

BedrockPlugin.h
3:class BedrockServer;
```

I don't really want to document a language feature in the code.